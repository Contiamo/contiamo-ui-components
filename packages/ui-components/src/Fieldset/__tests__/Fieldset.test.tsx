import * as React from "react"
import { render } from "enzyme"

import ThemelessFieldset from "../Fieldset"
import wrapDefaultTheme from "../../utils/wrap-default-theme"

const Fieldset = wrapDefaultTheme(ThemelessFieldset)

describe("Fieldset Component", () => {
  it("Should render", () => {
    const renderedComponent = render(<Fieldset value="SomeValue" />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
