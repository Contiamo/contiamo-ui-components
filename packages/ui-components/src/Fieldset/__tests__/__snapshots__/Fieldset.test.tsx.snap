// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fieldset Component Should render 1`] = `
<fieldset
  class="css-5u6dkf"
>
  <legend
    class="css-1l8opoi"
  />
</fieldset>
`;
